# ati netft
netft_ip_address: "192.168.1.1"
netft_sensor_name: "netft"
netft_frame_id: "end_effector"
netft_publish_rate: 1000
netft_print_flag: false
netft_file_path: "/home/yifanh/Git/catkin_ws/src/forcecontrol/data/netft_data.txt"

ati:
  offset: # will be added to the reading; in tool frame
    fx: -0.339213
    fy: 1.54215
    fz: -1.60007
    tx: -0.00325492
    ty: 0.0212634
    tz: 0.0168613
  gravity:
    x: 0.189769
    y: 0.236297
    z: -10.1566
  COM:
    x: -0.0180291
    y: 0.0175658
    z: 0.0370635
  safety:
    fx: 20 # maximum: 20
    fy: 20 # maximum: 20
    fz: 45 # maximum: 60
    tx: 1 # maximum: 1
    ty: 1 # maximum: 1
    tz: 1 # maximum: 1

# ABB EGM
egm_portnum: 6510
egm_max_dist_tran: 5 # mm, maximum distance between set point and current pose. 
egm_max_dist_rot: 0.05 # rad, maximum distance between set point and current pose. 
                      # This two numbers plus 'K' and bandwidth in robostudio determine 
                      # how fast the robot will move.
egm_safety_mode: 1 # 0: none 1: truncate 2: stop
egm_operation_mode: 0 # 0: cartesian 1: joint
egm_print_flag: false
egm_file_path: "/home/yifanh/Git/catkin_ws/src/forcecontrol/data/egm_data.txt"
egm_safety_zone:
  xmin: -50
  xmax: 220
  ymin: 240
  ymax: 550
  zmin: 260
  zmax: 600
  
# force control